t/routes-dataset.t

```diff
@@ -7,7 +7,7 @@ use Cro::HTTP::Test;
 use Test::Mock;
 use Test;
 
-plan 15;
+#plan 15;
```

Comment:
Can just remove it if relying on `done-testing`.

Reply:
In a way I prefer to set the plan.